Expecto/Expecto.fs

```diff
@@ -152,13 +152,13 @@ module Test =
 
 
 
-// TODO: make internal?
 module Impl =
   open Expecto.Logging
   open Expecto.Logging.Message
   open Helpers
   open Mono.Cecil
 
+  // TODO: remove mutable and also move the logger to the config!
   let mutable logger = Log.create "Expecto"
```

Comment:
Create a new Logger that can have its name changes & update usages of `Impl.logger.XYZ`